* enable parallel tests
* allow test scripts to inherit environment
* slightly raise tolerance in one test
author: Miguel Dias Costa (National University of Singapore)
--- Makefile.orig	2019-07-23 17:40:16.489733901 +0800
+++ Makefile	2019-07-23 17:40:33.656497133 +0800
@@ -157,7 +157,7 @@
 endif
 
 check: all
-	cd testsuite && $(MAKE) check
+	cd testsuite && $(MAKE) check && $(MAKE) check-parallel
 
 check-save: all
 	cd testsuite && $(MAKE) check-save
--- testsuite/run_testsuite.sh.orig	2019-07-25 14:48:12.000000000 +0800
+++ testsuite/run_testsuite.sh	2019-07-25 14:47:50.000000000 +0800
@@ -1,4 +1,4 @@
-#!/bin/bash -l
+#!/bin/bash
 #
 # Copyright (C) 2005-2009 Heiko Appel, David Strubbe
 #
--- MeanField/Utilities/mf_convert_wrapper.sh.orig	2019-07-26 09:52:27.648341000 +0800
+++ MeanField/Utilities/mf_convert_wrapper.sh	2019-07-26 09:52:39.922114228 +0800
@@ -1,4 +1,4 @@
-#!/bin/bash -l
+#!/bin/bash
 
 # David Strubbe, October 2010
 # Wrapper for mf_convert.x
--- testsuite/GaAs-EPM/GaAs.test.orig	2019-07-26 10:45:19.798520000 +0800
+++ testsuite/GaAs-EPM/GaAs.test	2019-07-26 10:45:37.753775275 +0800
@@ -32,7 +32,7 @@
 Output : WFN.out
 Input  : WFN.in PIPE
 
-Precision : 8e-15
+Precision : 9e-15
 match ; Eigenvalue  1 at k-pt 1   ; GREP(WFN.out,   "kpoint          1", 2,  1); -0.2710614199849328
 match ; Eigenvalue 10 at k-pt 1   ; GREP(WFN.out,   "kpoint          1", 2, 10);  1.2565373697755460
 match ; Eigenvalue 18 at k-pt 2   ; GREP(WFN.out,   "kpoint          2", 2, 18);  2.1322637363008994
